src/tests/ftest/osa/osa_offline_reintegration.py

```diff
@@ -24,111 +25,195 @@ def setUp(self):
         """Set up for test case."""
         super(OSAOfflineReintegration, self).setUp()
         self.dmg_command = self.get_dmg_command()
-        self.ior_apis = self.params.get("ior_api", '/run/ior/iorflags/*')
-        self.ior_test_sequence = self.params.get(
-            "ior_test_sequence", '/run/ior/iorflags/*')
-        self.ior_dfs_oclass = self.params.get(
-            "obj_class", '/run/ior/iorflags/*')
+        self.daos_command = DaosCommand(self.bin)
+        self.ior_test_sequence = self.params.get("ior_test_sequence",
+                                                 '/run/ior/iorflags/*')
+        self.test_oclass = self.params.get("oclass", '/run/test_obj_class/*')
+        self.loop_test_cnt = 1
         # Recreate the client hostfile without slots defined
         self.hostfile_clients = write_host_file(
             self.hostlist_clients, self.workdir, None)
+        self.dmg_command.exit_status_exception = True
 
     def run_offline_reintegration_test(self, num_pool, data=False,
-                                       server_boot=False):
+                                       server_boot=False, oclass=None,
+                                       reint_during_rebuild=False,
+                                       reint_during_aggregation=False):
```

Comment:
These can be defined inside osa_utils.py and used across all other OSA tests as needed.

Reply:
Ok. Will do.

Reply:
Done.